README.md

```diff
@@ -111,52 +113,52 @@ func BindingsOutput(ctx *ofctx.OpenFunctionContext, in []byte) ofctx.RetValue {
 }
 ```
 
-You can also use `ofn` to make a quick demo:
-
->By default the demo environment will be deleted when the demo finishes.
->You can keep the demo kind cluster for further exploration by `ofn demo --auto-prune=false`
->The demo kind cluster can be deleted by `kind delete cluster --name openfunction`
+You can also run the following command to make a quick demo:
 
 ```shell
 ofn demo
 ```
 
-For more information about ofn install, please refer to [ofn demo docs](https://github.com/OpenFunction/cli/blob/main/docs/demo.md).
+>By default, a demo environment will be deleted when a demo finishes.
+>You can keep the demo kind cluster for further exploration by running `ofn demo --auto-prune=false`.
+>The demo kind cluster can be deleted by running `kind delete cluster --name openfunction`.
+
+For more information about how to use the `ofn demo` command, refer to [ofn demo document](https://github.com/OpenFunction/cli/blob/main/docs/demo.md).
 
 ### Uninstall OpenFunction
 
-Use `ofn` to uninstall OpenFunction and its dependencies:
+Run the following command to uninstall OpenFunction and its dependencies.
 
 ```shell
 ofn uninstall --all
 ```
 
-For more information about ofn install, please refer to [ofn uninstall docs](https://github.com/OpenFunction/cli/blob/main/docs/uninstall.md).
+For more information about how to use the `ofn uninstall` command, refer to [ofn uninstall document](https://github.com/OpenFunction/cli/blob/main/docs/uninstall.md).
 
 ## 💻 Development
 
-You can get help on developing this project by visiting [Development Guide](docs/development/README.md).
+See the [Development Guide](docs/development/README.md) to get started with developing this project.
 
 ## 🛣️ Roadmap
 
-[Here](docs/roadmap.md) you can find OpenFunction's roadmap.
+Learn more about OpenFunction [roadmap](docs/roadmap.md).
 
 ## 🏘️ Community
 
 ### Community Call
 
-Meeting Info: [Zoom](https://us02web.zoom.us/j/89684762679?pwd=U1JNWVdzbElScVFMSEdQQnV0YnR4UT09)
+Meeting venue: [Zoom](https://us02web.zoom.us/j/89684762679?pwd=U1JNWVdzbElScVFMSEdQQnV0YnR4UT09)
```

Comment:
Is `venue` suitable for a virtual meeting?

Reply:
Dapr call it: Community Call : https://github.com/dapr/dapr or community meeting https://github.com/dapr/community#community-meetings

Keda call it Meeting Time under Community: https://github.com/kedacore/keda

Reply:
> Member

Right...I'll change it.

Reply:
> Dapr call it: Community Call : https://github.com/dapr/dapr or community meeting https://github.com/dapr/community#community-meetings
> 
> Keda call it Meeting Time under Community: https://github.com/kedacore/keda

Sure.

Reply:
[Updated](https://github.com/OpenFunction/OpenFunction/pull/195/commits/b9f8a5504b0356b6edfc9e33d1ab833f7467c7a2).